lib/versioner.js

```diff
@@ -3,6 +3,11 @@
 const _ = require('lodash');
 const createError = require('http-errors');
 const semver = require('semver');
+const {
+  compose, cond, curry,
+  flip, map, pipe, propEq, nAry, sort, toPairs, T } = require('ramda');
```

Comment:
You might be able to find corresponding methods in lodash which is already a dependency of this module.

Reply:
Yes, I'll do this now.